src/content/en/android/custom-tabs/implementation-guide/index.md

```diff
@@ -3,28 +3,29 @@ book_path: /web/android/_book.yaml
 description: Implementation Guide for Custom Tabs.
 
 {# wf_published_on: 2020-02-04 #}
-{# wf_updated_on: 2020-03-06 #}
+{# wf_updated_on: 2020-06-17 #}
 {# wf_blink_components: N/A #}
 
 # Custom Tabs Implementation guide {: .page-title }
 
-A complete example is available at [https://github.com/GoogleChrome/custom-tabs-client][1].It
+A complete example is available on [the GitHub sample application][1]. It
 contains re-usable classes to customize the UI, connect to the background service, and handle the
 lifecycle of both the application and the custom tab activity.
 
 If you follow the guidance from this page, you will be able to create a great integration.
 
-The first step for a Custom Tabs integration is adding the [Custom Tabs Support Library][2] to your
-project. Open your `build.gradle` file and add the support library to the dependency section.
+The first step for a Custom Tabs integration is adding the [AndroidX Browser Library][2] to your
+project. Open the `app/build.gradle` file and add the support library to the dependencies section.
 
 ```gradle
 dependencies {
     ...
-    compile 'com.android.support:customtabs:23.3.0'
+    implementation "androidx.browser:browser:1.2.0"
```

Comment:
Can you update this to match the styling of other docs such as https://developer.android.com/jetpack/androidx/releases/fragment
e.g.

```suggestion
    def androidx_browser_version = "1.2.0"
    
    implementation "androidx.browser:browser:$androidx_browser_version"
```

Reply:
This was actually taken directly from https://developer.android.com/jetpack/androidx/releases/browser

The libraries that define the variables do so because they have multiple `implementation` lines which use the same version. It's not the case for `androidx.browser`

Reply:
Ah excellent point!  I mistakenly thought that the `androidbrowserhelper` artifact was in the same group id, and shared a versioning strategy. 

Thanks!